src/main/resources/db/migration/V1_3__create_table_melding.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE MELDING
+(
+    id                              SERIAL PRIMARY KEY,
+    uuid                            CHAR(36) NOT NULL UNIQUE,
+    created_at                      TIMESTAMPTZ NOT NULL,
+    innkommende                     BOOLEAN NOT NULL,
```

Comment:
Betyr `innkommende=true` at den kom fra behandler?

Reply:
Ja, det var det jeg tenkte.

Reply:
Kanskje jeg går litt i sirkel nå, men hvis innkommende er et forståelig begrep her, kunne man kanskje brukt det overalt? Altså `incomingMeliding`/`outgoingMelding` e.l. i stedet for "til og fra behandler"? @juneih var vel inne på det samme med "sendt/mottatt"? 🤔